curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md

```diff
@@ -35,9 +35,20 @@ assert(
 The link's text should be `cat photos`. You have either omitted the text or have a typo.
 
 ```js
-const nestedAnchor = $('p > a')[0];
+const nestedAnchor = document.querySelector('p > a');
+const innerContent = nestedAnchor.innerHTML.toLowerCase();
 assert(
-  nestedAnchor.innerText.toLowerCase().replace(/\s+/g, ' ') === 'cat photos'
+	/^\s*cat photos\s*$/.test(innerContent)
+);
+```
+
+The link's text should be `cat photos`. You either have leading or ending whitespaces.
```

Comment:
This should still warn about changing the actual text. Not just about the whitespace.